config/config.json

```diff
@@ -18,6 +18,7 @@
     "password": null,
```

Comment:
Don't you need a password?

Reply:
the guide didn't specify if i needed one.  is there a default password i need to include?

Reply:
provide some password just to be on the safer side

Reply:
alright